HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -0,0 +1,184 @@
+
+subsection:: Choosing the device and the API
+list::
+##If you need to specify the device, you will need to do so for both input and output (setting both code::.inDevice::, as well as code::.outDevice::), unless you use ASIO
+##Both input and output device needs to use the same API (listed before the name, for example teletype::MME::, teletype::WDM-KS:: etc.).
+##Both input and output device should use the same sample rate (although some APIs might provide resampling). This can be set in Windows's Audio Control Panel.
+::
+The following list provides basic reference for different APIs. The most recommended APIs are listed first.
+
+Multichannel operation refers to using more than 2 inputs/outputs simultaneously.
+
+subsection:: List of available APIs
+definitionlist::
+## ASIO
+||
+LIST::
+## strong::Multichannel operation: :: Yes, if supported by the hardware
+## strong::Typical latency: :: Low
+## strong::Caveats: :: Dedicated ASIO driver needs to be provided by the audio device's manufacturer; this is common for most (semi-)professional interfaces, but not necessarily for internal soundcards
+## strong::More info: :: Designed for pro-audio devices; control over sampling rate and hardware buffer size is usually provided by the driver's control panel (i.e. not settable by SuperCollider); ASIO stands for Audio Stream Input/Output and was developed by Steinberg
+::
+## WASAPI
+||
+LIST::
+## strong::Multichannel operation: :: Usually no
+## strong::Typical latency: :: Low
+## strong::Caveats: :: If the requested sample rate is different than the device's sample rate, the server will not boot
+## strong::More info: :: WASAPI stands for Windows Audio Session API and is the most modern Windows audio API
+::
+## WDM-KS
+||
+LIST::
+## strong::Multichannel operation: :: Usually no
+## strong::Typical latency: :: Low
+## strong::Caveats: :: On some systems SuperCollider will prevent other applications from using the audio device when using this API
+## strong::More info: :: WDM-KS stands for Windows Driver Model Kernel Streaming, it was the first native Windows API providing reasonably low latency
+::
+## DirectSound
+||
+LIST::
+## strong::Multichannel operation: :: Usually no
+## strong::Typical latency: :: Moderate/high
+## strong::Caveats: :: It is an older API and typically provides worse performance than the newer ones
+## strong::More info: :: DirectSound is part of DirectX and was originally created with game audio in mind
+::
+## MME
+||
+LIST::
+## strong::Multichannel operation: :: Usually no
+## strong::Typical latency: :: High
+## strong::Caveats: :: It is the oldest API on this list; it is chosen by default if the user does not specify which device to use
+## strong::More info: :: This API might work out-of-the-box, but choosing a newer one instead will usually provide better performance; MME stands for Multimedia Extension (for Windows 3.0)
+::
+::
```

Comment:
would this work better as a table?

Reply:
Nope, I tried this. It was not looking good as I was not able to control the table's spacing. Unfortunately the original commits were rebased so I can't demonstrate easily.

Reply:
ok! I believe you :)